# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: media/ffvpx/libavcodec/rl.h
# Commit: 186e2a334f2d
# Full Hash: 186e2a334f2deb37b29f44b28bc8c83babeeb9a4
# Author: Paul Adenot <paul@paul.cx>
# Date: 2024-02-09 09:37:17
# Regressor Bug: 1877221
# File Overlap Count: 1
# Description:
#   Bug 1877221 - Pull in new ffmpeg code, reapply patch. r=media-playback-reviewers,chunmin
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D199909
# ==============================================================================

diff -r c5920de78d16 -r 186e2a334f2d media/ffvpx/libavcodec/rl.h
--- a/media/ffvpx/libavcodec/rl.h	Thu Feb 08 10:39:53 2024 +0000
+++ b/media/ffvpx/libavcodec/rl.h	Thu Feb 08 10:39:54 2024 +0000
@@ -80,7 +80,7 @@
  */
 void ff_rl_init_vlc(RLTable *rl, unsigned static_size);
 
-#define INIT_VLC_RL(rl, static_size)\
+#define VLC_INIT_RL(rl, static_size)\
 {\
     static RL_VLC_ELEM rl_vlc_table[32][static_size];\
 \